schema/oneof_test.go

```diff
@@ -140,68 +127,94 @@ var oneOfTestCMappedSchema = schema.NewStructMappedObjectSchema[oneOfTestObjectC
 	oneOfTestObjectCProperties,
 )
 
-var oneOfTestInlineObjectBProperties = map[string]*schema.PropertySchema{
-	"message": schema.NewPropertySchema(
-		schema.NewStringSchema(nil, nil, nil),
-		nil,
-		true,
-		nil,
-		nil,
-		nil,
-		nil,
-		nil,
-	),
-	"choice": schema.NewPropertySchema(
-		schema.NewStringSchema(nil, nil, nil),
-		nil,
-		true,
-		nil,
-		nil,
-		nil,
-		nil,
-		nil,
-	),
-}
+// Test_OneOf_ConstructorBypass tests the behavior of a OneOf object created
+// by the Scope Scope Schema through unserialization of data without using a
```

Comment:
"Scope Scope Schema"?

Reply:
@jaredoconnell would need to explain that one

Reply:
It's a scope that contains the schema of a scope and an object.